### DISCUSSION TITLE
Mayday, unable to run homebrew?

### DISCUSSION THREAD

**Discussion by itcodes** [Homebrew/discussions#810] [Getting started]
Today, I bought a new MAC for M1. After installing homebrew, I found some problems in operation. I want to ask you how to solve them
![image](https://user-images.githubusercontent.com/8282645/107878991-3dfc3c80-6f11-11eb-9b53-ac44a7467b08.png)

**Comment by itcodes** [comment]
Thank you for finding the historical problem and solving it. Thank you very much.
https://github.com/Homebrew/discussions/discussions/775

**Comment by carlocab** [comment]
Did you run the commands at the end of the install telling you to add `brew` to your `PATH`?

**Reply by itcodes** [reply]
I forgot to add the path, it's my problem. It's been solved.
